open_academy/views/open_academy_course_views.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="open_academy_course_action" model="ir.actions.act_window">
+        <field name="name">Courses</field>
+        <field name="res_model">open.academy.course</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+
+    <record id="open_academy_course_tree" model="ir.ui.view">
+        <field name="model">open.academy.course</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="name" string="Courses"/>
+                <field name="description" string="Description"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="open_academy_course_search" model="ir.ui.view">
```

Comment:
Same as above. See the [guidelines](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#25naming-xml_id) for naming xml_id